src/global.d.ts

```diff
@@ -0,0 +1,38 @@
+import * as PIXI from 'pixi.js';
+import GestureEvent from './gesture-event';
+
+declare global {
+  interface GestureEventMap extends HTMLElementEventMap {
+    gesturestart: GestureEvent;
+    gesturechange: GestureEvent;
+    gestureend: GestureEvent;
+  }
+
+  interface HTMLElement {
+    addEventListener<K extends keyof GestureEventMap>(
+      eventName: K,
+      listener: (
+        this: HTMLElement,
+        ev: GestureEventMap[K],
+        options?: boolean | AddEventListenerOptions
+      ) => any
```

Comment:
Could/should these be void?

Reply:
I was thinking that, but I followed what `lib.dom.d.ts` has for the existing declaration here.

See: https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L6711